scripts/run_scheduler.py

```diff
@@ -158,9 +160,24 @@
     # Note that the actual scores are generated using the Ranker (components.ranker.__init__.py, Ranker class), which
     # follows the old implementation but is generalized to multi-night, and cleaned up significantly.
 
-
     # Sergio preliminary work:
     # Output the data in a spreadsheet.
     # for program in collector._programs.values():
     #    if program.id == 'GS-2018B-Q-101':
     #        atoms_to_sheet(program)
+
+    # "Selection" format for optimzer
+    # Dict[GroupID, Tuple[Group, GroupInfo]
+    selection = {}
+    for program_id, group_info_map in results.items():
+        for group_id, group_info in group_info_map.items():
+            group = selector.get_group(group_id)
+            selection[group_id] = (group, group_info)  # TODO: We might consider just add group info to group?
```

Comment:
This should probably be moved directly to the Selector. I can do that, but leave it here now so you can keep working on this until I've done that.